p2p/gossip_test.go

```diff
@@ -0,0 +1,90 @@
+package p2p
```

Comment:
this file was removed.

Reply:
Removed e2621f417a474187c68fac1242d5e974ec561ff2.